src/noUninitializedRule.ts

```diff
@@ -39,7 +39,18 @@ class NoUninitializedPropertiesWalker extends Lint.RuleWalker {
 
     private _initializedProperties: string[][] = [];
 
-    visitClassDeclaration(node: ts.ClassDeclaration) {
+    visitClassLikeDeclaration(
+        node: ts.ClassDeclaration,
+        superMethod: Lint.RuleWalker['visitClassDeclaration'],
+    ): void;
+    visitClassLikeDeclaration(
+        node: ts.ClassExpression,
+        superMethod: Lint.RuleWalker['visitClassExpression'],
+    ): void;
+    visitClassLikeDeclaration(
+        node: ts.ClassDeclaration | ts.ClassExpression,
+        superMethod: Lint.RuleWalker['visitClassDeclaration'] | Lint.RuleWalker['visitClassExpression'],
```

Comment:
This could be simplified to accept the name of the super method, so that the implementation of this method can guarantee that a super method is actually called.